jib-build-plan/CHANGELOG.md

```diff
@@ -5,9 +5,17 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
-- Added file ownership information in `FileEntry` and `FileEntriesLayer`. ([#2494](https://github.com/GoogleContainerTools/jib/pull/2494))
+- Added `Platform` class representing an image platform. ([#2584](https://github.com/GoogleContainerTools/jib/pull/2584))
+- Added `get/setPlatforms()` and `addPlatform()` to `ContainerBuildPlan` for setting and getting image platforms. ([#2584](https://github.com/GoogleContainerTools/jib/pull/2584))
 
 ### Changed
 
+- Removed `get/setOsHint()` and `get/setArchitectureHint()` in faver of `get/setPlatforms()` and `addPlatform()`. ([#2584](https://github.com/GoogleContainerTools/jib/pull/2584))
+
 ### Fixed
 
+## 0.2.0
```

Comment:
We forgot to update CHANGELOG when we released 0.2.0, so doing this now.

Reply:
We edit the ` CHANGELOG ` when we make updates to the api only right?

Reply:
Yes. `CHANGELOG` is to highlight user-visible changes, so that they understand what has changed between releases.